benchmark_runner/workloads/vdbench_pod.py

```diff
@@ -41,22 +62,51 @@ def run(self):
             else:
                 self.__es_index = 'vdbench-results'
             self._environment_variables_dict['kind'] = self.__kind
-            self._oc.create_pod_sync(yaml=os.path.join(f'{self._run_artifacts_path}', f'{self.__name}.yaml'), pod_name=self.__pod_name)
-            self._oc.wait_for_initialized(label=f'app=vdbench-{self._trunc_uuid}', label_uuid=False)
-            self._oc.wait_for_ready(label=f'app=vdbench-{self._trunc_uuid}', label_uuid=False)
-            self.__status = self._oc.wait_for_pod_completed(label=f'app=vdbench-{self._trunc_uuid}', label_uuid=False, job=False)
-            self.__status = 'complete' if self.__status else 'failed'
-            # save run artifacts logs
-            result_list = self._create_pod_run_artifacts(pod_name=self.__pod_name)
-            if self._es_host:
-                # upload several run results
-                for result in result_list:
-                    self._upload_to_elasticsearch(index=self.__es_index, kind=self.__kind, status=self.__status, result=result)
-                # verify that data upload to elastic search according to unique uuid
-                self._verify_elasticsearch_data_uploaded(index=self.__es_index, uuid=self._uuid)
-            self._oc.delete_pod_sync(
-                yaml=os.path.join(f'{self._run_artifacts_path}', f'{self.__name}.yaml'),
-                pod_name=self.__pod_name)
+            if not self._scale:
+                self._oc.create_pod_sync(yaml=os.path.join(f'{self._run_artifacts_path}', f'{self.__name}.yaml'), pod_name=self.__pod_name)
+                self._oc.wait_for_initialized(label=f'app=vdbench-{self._trunc_uuid}', label_uuid=False)
+                self._oc.wait_for_ready(label=f'app=vdbench-{self._trunc_uuid}', label_uuid=False)
+                self.__status = self._oc.wait_for_pod_completed(label=f'app=vdbench-{self._trunc_uuid}', label_uuid=False, job=False)
+                self.__status = 'complete' if self.__status else 'failed'
+                # save run artifacts logs
+                result_list = self._create_pod_run_artifacts(pod_name=self.__pod_name)
+                if self._es_host:
+                    # upload several run results
+                    for result in result_list:
+                        self._upload_to_elasticsearch(index=self.__es_index, kind=self.__kind, status=self.__status, result=result)
+                    # verify that data upload to elastic search according to unique uuid
+                    self._verify_elasticsearch_data_uploaded(index=self.__es_index, uuid=self._uuid)
+                self._oc.delete_pod_sync(
+                    yaml=os.path.join(f'{self._run_artifacts_path}', f'{self.__name}.yaml'),
+                    pod_name=self.__pod_name)
+            # scale
+            else:
+                # create redis and state signals
+                sync_pods = {'redis': 'redis', 'state_signals_exporter_pod': 'state-signals-exporter'}
```

Comment:
Use redis-master as the name for the sync pod; no need for the extra logic at lines 86-90.

Reply:
Fixed